client/app/scripts/superdesk-monitoring/aggregate.js

```diff
@@ -489,7 +489,7 @@
                     if (scope.privateSavedSearches.length > 0) {
                         scope.privateSavedSearches.length = 0;
                     }
-                    _.each(scope.searches, function(item) {
+                    _.each(_.sortBy(scope.searches, 'name'), function(item) {
```

Comment:
We are using `sortByName` filter for this kind of functionality. Can we use the same to be consistent with other lists also?